### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Hegabovic**
added style and fonts

### CODE REVIEW

**Review by MohamedEldefrawy — APPROVED** [review]
Great work

File: Static/css/style.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+:root {
+    --backGround:rgb(212, 225, 233);
+    --paragraphs:rgb(36, 47, 50);
+    --logOut-logIn-purchase:rgb(249, 174, 58);
+    --navBar:rgb(73, 101, 113);
+
```
Comment (MohamedEldefrawy) [inline Static/css/style.css:6]: Remove white spaces

**Review by Muhammed-Ismail2017 — COMMENTED** [review]
approved

**Review by MohamedEldefrawy — APPROVED** [review]
Great work

**Pull request merged**